include/gz/rendering/base/BaseGizmoVisual.hh

```diff
@@ -495,6 +495,8 @@ namespace ignition
       this->visuals[TransformAxis::TA_TRANSLATION_X] = transXVis;
       this->visuals[TransformAxis::TA_TRANSLATION_Y] = transYVis;
       this->visuals[TransformAxis::TA_TRANSLATION_Z] = transZVis;
+      // Store the translation origin visual in this->visuals using a key
```

Comment:
As a user of `GizmoVisual`, it's difficult to know how to access the origin without reading the code. I would suggest adding something in `ChildByAxis` or `TransformAxis` itself.

Reply:
There is not a enum for it in `TransformAxis` or accessible by `ChildByAxis` because it's currently not meant to be exposed to the public. So checking to see what the use case is for accessing the GizmoVisual's origin visual? Do you need to change the origin's visual properties like color or size? If we want to expose this, we'll need to target `main`.

Reply:
Actually, that gz-sim PR accesses the visual in `TA_ROTATION_Z << 1`, which is a gray circle that has to be rotated just like the other rotation circles, and so might need to be public. 

This is also done in `TransformController`: https://github.com/gazebosim/gz-rendering/blob/e58870ce3ea82711fc42777709e2170341b1eb3c/src/TransformController.cc#L161-L162

The origin visual referenced by `TA_TRANSLATION_Z << 1` isn't used in those cases.

Reply:
as per https://github.com/gazebosim/gz-sim/pull/2051#discussion_r1296364794, I'll do some refactoring of code so we can avoid exposing the confusing `TransformAxis::TA_ROTATION_Z << 1` value